sqldelight-gradle-plugin/src/test/fixtures/keyword_identifiers/src/main/sqldelight/com/sample/Test.sq

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE test (
+  "DESC" TEXT,
+  `DESC` TEXT,
+  [DESC] TEXT AS java.util.List
```

Comment:
no. our gradle plugin isn't part of javac like apt so we don't know anything about classes